fuzz/src/full_stack.rs

```diff
@@ -401,7 +401,7 @@ pub fn do_test(data: &[u8], logger: &Arc<dyn Logger>) {
 				sha.input(&payment_hash.0[..]);
 				payment_hash.0 = Sha256::from_engine(sha).into_inner();
 				payments_sent += 1;
-				match channelmanager.send_payment(route, payment_hash) {
+				match channelmanager.send_payment(route, payment_hash, &None) {
```

Comment:
Yea, likely, I'm not too worried about payment_secret coverage, there aren't that many ways it can go wrong, and we hit a number of them. As we write new tests, including payment_secrets in them is likely a good idea, but going to each test and setting payment_secrets requires some writing/reviewing effort to make sure it all lines up right.